src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+
```

Comment:
could make it even shorter
```suggestion
    name,
    version,
    wheels,

```

Reply:
Agree